.rubocop.yml

```diff
@@ -17,6 +17,7 @@ AllCops:
     - 'Gemfile.lock'
     - 'config.ru'
     - 'db/migrate/20220120101655_create_active_storage_tables.active_storage.rb'
+    - 'db/migrate/20220128234636_devise_token_auth_create_users.rb'
```

Comment:
Specific files should be excluded in `.rubocop_todo.yml`
https://docs.rubocop.org/rubocop/configuration.html#automatically-generated-configuration